Sources/Plasma/FeatureLib/pfPython/cyAccountManagement.cpp

```diff
@@ -134,11 +128,6 @@ bool cyAccountManagement::IsActivePlayerSet()
     return NetCommGetPlayer()->playerInt != 0;
 }
 
-void cyAccountManagement::UpgradeVisitorToExplorer(unsigned playerId)
-{
-    NetCommUpgradeVisitorToExplorer(playerId, nil);
```

Comment:
This deletion orphans the function `NetCommUpgradeVisitorToExplorer`. I can't help but wonder if we want to remove that as well? Of course, if we do that, we then orphan `NetCliAuthUpgradeVisitorRequest` and its transaction. Comments from the gallery, @Deledrius, @zrax?